pkg/apiserver/rest/common.go

```diff
@@ -33,3 +39,43 @@ func sendError(ctx echo.Context, code int, message string) error {
 func sendResponse(ctx echo.Context, code int, object interface{}) error {
 	return ctx.JSON(code, object)
 }
+
+func (s *ServerImpl) GetOpenAPISpec(ctx echo.Context) error {
+	swagger, err := server.GetSwagger()
+	if err != nil {
+		return fmt.Errorf("failed to load swagger spec: %v", err)
+	}
+
+	// Use the X-Forwarded-* headers to populate the OpenAPI spec with the
+	// location where the API is being served. Using this trick the
+	// swagger-ui service dynamically loads the OpenAPI spec from the
+	// APIServer and knows where to send the TryItNow requests.
+	// Wherever the API server is accessed from through a proxy or
+	// sub-domain this will correct the servers entry to match that clients
+	// access path.
+	headers := ctx.Request().Header
+
+	log.Debugf("Got headers %#v", headers)
+
+	serverurl := &url.URL{}
```

Comment:
```suggestion
	serverURL := &url.URL{}
```

Reply:
I'm surprised that the linter didn't pick up on this honestly, but I'm going to leave this for now to save any further delays on this PR.